### DISCUSSION TITLE
Excluding certain parameters when adopting solutions

### DISCUSSION THREAD

**Discussion by nbungart1999** [phoebe-project/phoebe2#488] [Questions]
When adopting solutions such as ebai_sol, how do I not adopt certain parameters such as the inclination?

**Comment by nbungart1999** [comment]
> When adopting solutions such as ebai_sol, how do I not adopt certain parameters such as the inclination?

Because looking at one of the tutorials all they do is flip constraints that ebai wasn't good at

**Reply by kecnry** [reply]
See the `adopt_parameters` parameter in the solution.  You can either change the value of that parameter or pass as a keyword argument override to [b.adopt_solution](http://phoebe-project.org/docs/latest/api/phoebe.frontend.bundle.Bundle.adopt_solution)
